compiler+runtime/include/cpp/jank/read/lex.hpp

```diff
@@ -151,14 +151,19 @@ namespace jank::read::lex
     processor(native_persistent_string_view const &f);
 
     result<token, error> next();
-    result<codepoint, error> peek() const;
+    result<codepoint, error> peek(native_integer const ahead = 1) const;
     option<error> check_whitespace(native_bool const found_space);
-    
+    native_bool is_valid_num_char(char const c) const;
+
```

Comment:
This doesn't need to be a member fn, from what I can see. It can just be a static fn in the cpp file.

Reply:
udpated.